docs/changelog.rst

```diff
@@ -38,7 +38,7 @@ Minor Changes:
     - Updated the example notebooks and created a quickstart notebook.
     - Fixed broken links in the documentation.
     - Add legend to SilhouetteVisualizer for the average score axis
-    - Move SilhouetteVisualizer cluster identity labels from the middle to the y-axis.
```

Comment:
It looks like this line somehow disappeared @pdamodaran &mdash; can you double check?

Reply:
The line was indeed missing in the latest commit - I will add it in.